adhocracy4/comments_async/static/comments_async/comment_form.jsx

```diff
@@ -87,6 +87,18 @@ export default class CommentForm extends React.Component {
   render () {
     const textareaStyle = { height: (this.state.textareaHeight) + 'px' }
     const hasParent = this.props.parentIndex !== undefined
+    const actionValue = this.props.commentCategoryChoices
+      ? translated.post
+      : translated.comment
```

Comment:
@khamui I know you did't introduce this, but do you know what it is used for? I mena, the different action value?

Reply:
not really sure, i will double check that

Reply:
Yes... but why? Is that maybe just some leftover? Could you try and take it out and check if it still works (with and without categories)?

Reply:
hm, do you think it is enough to verify? because taking it out here, can break in another project where it might be used?

So if the flag `withCategories` is false:
<img width="418" alt="Bildschirmfoto 2022-06-07 um 12 05 38" src="https://user-images.githubusercontent.com/52801593/172364970-6cda107c-66a5-4dcd-9a2d-5efd77f66777.png">

if its true:
<img width="484" alt="Bildschirmfoto 2022-06-07 um 12 04 39" src="https://user-images.githubusercontent.com/52801593/172365014-37057a53-ba37-4e36-911f-7f792352b69a.png">

I don't know when this flag is set (or in which projects?). But i am hesitating taking this out and just verify in aplus.

Reply:
Well the code for the comments is all in A4, so it should be fine to test it in one project..
I just don't get why the value of the button changes depending on `withCategories`? But anyway, I can also check that!